### PR TITLE
AWS Inspector

### PR DESCRIPTION
**Pull request by MattiasGees**
This configures AWS inspector with the rules and tags you want. This will also run on a regular basis with Cloudwatch rules.

The `rules_package_arns` will disappear soon as I am working on implementing a datasource to get the arns for the specific region.

### CODE REVIEW

File: aws-inspector/iam.tf [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "inspector" {
+  name = "cloudwatch-event-inspector-${var.name}"
```
Comment (SamClinckspoor) [inline aws-inspector/iam.tf]: can this be `name_prefix `?

Comment (MattiasGees) [reply aws-inspector/iam.tf]: So implement an extra variable?

Comment (SamClinckspoor) [reply aws-inspector/iam.tf]: how do you mean an extra var?
your `aws_iam_role` also uses `name_prefix` btw

Comment (iuriaranda) [reply aws-inspector/iam.tf]: Yeah I also prefer to use `name_prefix` whenever possible. It avoids extra headaches in the future :)
You don't need an extra var for that btw

**Pull request merged**